packages/design-system/src/components/modal-button/ModalButton.stories.tsx

```
@@ -0,0 +1,70 @@
+export const RightButton: Story = {
+  args: {
+    text: 'Button',
+    variant: 'right',
+    onClick: () => console.log('Right button clicked'),
+  },
+};
+
+export const ButtonWrapper: Story = {
+  render: () => (
+    <ModalButtonWrapper
+      children={[
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)